kotlin-desktop-toolkit-linux/src/main/kotlin/org/jetbrains/desktop/linux/Application.kt

```diff
@@ -0,0 +1,324 @@
+package org.jetbrains.desktop.linux
```

Comment:
Why did you decide to put in a separate project? (`kotlin-desktop-toolkit-linux` vs. the existing `kotlin-desktop-toolkit`) Asking to know if I should do the same 😊

Reply:
You should really use variants and publications under the same project instead, it's not harder than different projects, but much easier administratively for the build. Subprojects makes sharing things much harder, and Gradle mistakes easier to do.